CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -13,34 +13,69 @@
 import SwiftSyntax
 
 public struct KeywordSpec {
-  public var name: String
+  /// The name of the keyword.
+  public let name: String
 
-  /// If `true`, this is for an experimental language feature, and any public
-  /// API generated should be SPI.
-  public var isExperimental: Bool
+  /// Indicates if the keyword is part of an experimental language feature.
+  ///
+  /// If `true`, this keyword is for an experimental language feature, and any public
+  /// API generated should be marked as SPI
+  public let isExperimental: Bool
 
-  public var isLexerClassified: Bool
-  public var requiresLeadingSpace: Bool
-  public var requiresTrailingSpace: Bool
+  /// Indicates if the token kind is switched from being an identifier to a keyword in the lexer.
+  public let isLexerClassified: Bool
 
-  public var escapedName: String {
+  /// Indicates if the keyword requires a leading space.
+  ///
+  /// If `true`, the keyword should have a space character before it when used in Swift code.
+  public let requiresLeadingSpace: Bool
+
+  /// Indicates if the keyword requires a trailing space.
+  ///
+  /// If `true`, the keyword should have a space character after it when used in Swift code.
+  public let requiresTrailingSpace: Bool
```

Comment:
I think we no longer use `requiresLeadingSpace` and `requiresTrailingSpace` to generate any code. You should be able to just remove them. You can also do it in a follow-up PR if you want.

Reply:
I've removed both properties in one of the commits in this PR. Thanks for catching that!

Reply:
And by the way @ahoppen do you prefer splitting commits like I've done there or all changes should be in one commit?

Reply:
I think if they are more or less independent changes, having them in separate commits is good 👍🏽